diff -up ImageMagick-6.9.13-17/configure.ac.2~ ImageMagick-6.9.13-17/configure.ac
--- ImageMagick-6.9.13-17/configure.ac.2~	2024-10-25 23:47:11.873120946 +0200
+++ ImageMagick-6.9.13-17/configure.ac	2024-10-25 23:48:23.193908889 +0200
@@ -1263,13 +1263,13 @@ if test "${OPENMP_ENABLED}" = 'yes' && \
     if test "x$GOMP_LIBS" = x ; then
       if $CC --version 2>&1 | grep clang > /dev/null ; then
         AC_COMPILE_IFELSE([ AC_LANG_PROGRAM([#include <omp.h>], [
-          omp_set_num_threads(1);])],[GOMP_LIBS="-lomp"],[])
+          omp_set_num_threads(1);])],[GOMP_LIBS="-fopenmp"],[])
       fi
     fi
     # GCC
     if test "x$GOMP_LIBS" = x ; then
       AC_COMPILE_IFELSE([ AC_LANG_PROGRAM([#include <omp.h>], [
-        omp_set_num_threads(1);])],[GOMP_LIBS="-lgomp"],[])
+        omp_set_num_threads(1);])],[GOMP_LIBS="-fopenmp"],[])
     fi
   else
     # Sun CC
