diff -up ImageMagick6-6.9.12-2/configure.ac.5~ ImageMagick6-6.9.12-2/configure.ac
--- ImageMagick6-6.9.12-2/configure.ac.5~	2021-03-04 14:16:40.354698741 +0100
+++ ImageMagick6-6.9.12-2/configure.ac	2021-03-04 14:17:09.265765722 +0100
@@ -1885,7 +1885,7 @@ if test "$with_fpx" != 'no'; then
             AC_MSG_RESULT([no -- some components failed test])
             have_fpx='no (failed tests)'
         else
-            FPX_LIBS='-lfpx'
+            FPX_LIBS='-lfpx -lstdc++'
             AC_DEFINE([FPX_DELEGATE],[1],[Define if you have FlashPIX library])
             AC_MSG_RESULT([yes])
             have_fpx='yes'
